.azure/pipelines/helix-matrix.yml

```diff
@@ -31,14 +39,17 @@ jobs:
     agentOs: Windows
     timeoutInMinutes: 300
     steps:
+    - ${{ if eq(variables['Build.CronSchedule.DisplayName'], 'IdentityModel test job') }}:
+      - script: "echo ##vso[build.addbuildtag]identitymodel-test"
```

Comment:
Shouldn't it be this?
```suggestion
      - script: echo "##vso[build.addbuildtag]identitymodel-test"
```

Reply:
This is how we do it elsewhere: https://github.com/dotnet/aspnetcore/blob/d3f5b217e5b54df6d6897b16c3ad86efd03496d9/.azure/pipelines/ci.yml#L176-L178

https://dev.azure.com/dnceng-public/public/_build?definitionId=83&tagFilter=daily-build

Reply:
Bizarre... This is how dngecng folks been fixing the arcade:

![image](https://github.com/dotnet/aspnetcore/assets/4403806/165b2f19-9153-463a-9c5a-bdaa739bb1bb)

Reply:
Ah, looks like this approach works on Windows, but not Mac/Linux: https://github.com/dotnet/arcade/pull/14687. I'll put up a PR to fix our usage - thanks!